Example/SignUp/SignUpPresetner.swift

```diff
@@ -50,6 +50,16 @@ final class SignUpPresenter {
         navigator?.showAlert(title: "Did sign up", message: "\(state)")
     }
 
+    /// - note: Simulates asynchronous picker presentation
```

Comment:
Why do we need this comment? 1) It's the example app. 2) At line 57 it's obvious that it's async. 🤔

Reply:
Example code will be deleted after having some reviews.
I need to clarify my impl is the right approach first.